seed/challenges/02-data-visualization-certification/sass.json

```diff
@@ -1,19 +1,307 @@
+        "Making variables become exteremly useful when we want similar elemnts to have the same color, for example. If wanted to change the color to maybe a slightly different shade, all we would have to do is change the variables value.",
+        "<h4>Instructions</h4>",
+        "Create a variable <code>$text-color</code> and set it equal to <code>red</code>",
+        "Then change the value of all <code>color</code> properties by using <code>color: $text-color</code>.",
+        "Finally, change the value of <code>$text-color</code> to another color and see how all respective elements change color!"
+      ],
+      "challengeSeed": [
+        "<style>",
+        "  /*Only change code below this line.*/",
+        "  ",
+        "  /*Only change code below this line.*/",
+        "  .header{",
+        "    text-align: center;",
+        "  }",
+        "  .blog-post h3{",
+        "  /*Only change code below this line.*/",
+        "    color: red;",
+        "  /*Only change code below this line.*/",
+        "  }",
+        "</style>",
+        "",
+        "<h1 class=\"header\">Lern Sass</h1>",
+        "<div class=\"blog-post\">",
+        "  <h3>Some random title</h3>",
+        "  <p>This is a pargraph with some very random text in it</p>",
+        "</div>",
+        "<div class=\"blog-post\">",
+        "  <h3>Header #2</h3>",
+        "  <p>Here is some more random text.</p>",
+        "</div>",
+        "<div class=\"blog-post\">",
+        "  <h3>Here is another header</h3>",
+        "  <p>Even more random text within a paragraph</p>",
+        "</div>"
+      ],
+      "tests": [],
+      "type": "waypoint",
+      "challengeType": 0
+    },
+    {
+      "id": "5690af775b3908042c47b329",
+      "title": "Nesting your CSS",
+      "description": [
+        "<code>Nesting</code> is a really useful way of organising your CSS.",
+        "Normally, you would target each element on a different line to style it, like so:",
+        "<blockquote>nav{<br>&nbsp;&nbsp;background-color: red;<br>}<br><br>nav ul{<br>&nbsp;&nbsp;list-style: none;<br>}<br><br>li{<br>&nbsp;&nbsp;display: inline-block;<br>}</blockquote>",
+        "As you can imagine, in larger projects, your CSS file will get pretty messy. Here is where <code>nesting</code> comes in - it allows you to organise your different styles by nesting them within their respective parent elements:",
+        "<blockquote>nav{<br>&nbsp;&nbsp;background-color: red;<br><br>&nbsp;&nbsp;ul{<br>&nbsp;&nbsp;&nbsp;&nbsp;list-style: none;<br><br>&nbsp;&nbsp;&nbsp;&nbsp;li{<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;display: inline-block;<br>&nbsp;&nbsp;&nbsp;&nbsp;}<br>&nbsp;&nbsp;}<br>}<br></blockquote>",
+        "<h4>Instructions</h4>",
+        "Use the <code>nesting</code> technique shown above to style both children of <code>.blog-post</code>. The <code>h1</code> should be centered and be blue. The <code>p</code> should have a font size of 20px."
```

Comment:
~~The <code>h1</code> should be centered and be blue. The <code>p</code> should have a font size of 20px.~~ The <code>h1</code> **tag** should be **centred** and be blue. The <code>p</code> **tag** should have a font size of 20px.